cli/module_loader.rs

```diff
@@ -40,6 +56,65 @@ impl CliModuleLoader {
       ps,
     })
   }
+
+  fn load_prepared_module(
```

Comment:
I had initially moved this code from proc_state here in the previous PR because I couldn't store an sqlite connection in an Arc, but I kind of like it here better anyway, so I left this here for this PR.